app/controllers/monitoring_controller.rb

```diff
@@ -1,35 +1,43 @@
-class MonitoringController < ActionController::Base
+class MonitoringController < ApplicationController
+  skip_before_action :set_feature_flag_users
```

Comment:
we should also skip the set sentry user as it hits the database as well (tested on migration)

Reply:
and also the session as it's not needed for the healthcheck